apps/frontend/src/app/PageCharacter/CustomMultiTarget.tsx

```diff
@@ -261,17 +264,12 @@ function CustomMultiTargetDisplay({ index, target, setTarget, expanded, onExpand
           color={target.targets.length ? "success" : undefined}
           label={<Trans t={t} i18nKey="multiTarget.target" count={target.targets.length}>{{ count: target.targets.length }} Targets</Trans>}
         />
-        <StyledInputBase
-          value={name}
-          sx={{ borderRadius: 1, px: 1, flexGrow: 1 }}
-          onChange={(e: ChangeEvent<HTMLInputElement>) => setName(e.target.value)}
-          onBlur={saveName}
-        />
+        <NameTextField name={target.name} setName={saveName} />
         {target.description && <InfoTooltip title={
           <Typography>
-            {target.description.length > 315 ? target.description.slice(0, 300) + "..." : target.description}
+            {target.description.length > MAX_DESC_TOOLTIP_LENGTH ? target.description.slice(0, MAX_DESC_TOOLTIP_LENGTH) + "..." : target.description}
```

Comment:
I added a bit extra to the check so it doesn't ellipse if theres just a little extra to show anyways. Up to you if you want to keep that

Reply:
might as well, you can argue for like 301 length would add a 3 ellipse to be 303 character, which is kinda weird, but it honestly doesn't matter.